src/cljs/main/broadfcui/main.cljs

```diff
@@ -175,9 +175,32 @@
    (fn []
      {:user-status #{}})
    :component-will-mount
-   (fn [{:keys [this]}]
+   (fn [{:keys [this state]}]
      (init-nav-paths)
-     (this :handle-hash-change))
+     (this :handle-hash-change)
+     (set! js/window.forceSignedIn
+           (fn [auth-token]
+             (utils/ajax {:url (str "https://www.googleapis.com/oauth2/v3/tokeninfo?access_token=" auth-token)
+                          :on-done
+                          (fn [{:keys [status-code success? get-parsed-response]}]
+                            (if success?
+                              (let [{:keys [email sub]} (get-parsed-response)
+                                    instance (clj->js
+                                              {:currentUser
+                                               {:get
+                                                (fn []
+                                                  (clj->js {:getAuthResponse
+                                                            (fn [] (clj->js {:access_token auth-token}))
```

Comment:
`(fn [] (clj->js {:access_token auth-token}))` -> `(constantly (clj->js {:access_token auth-token}))`

Reply:
I will make this change if this is more idiomatic. However, while I guess I like the intent declared in words, I am confused by the preference for words over shorter symbolic idioms that seem more typical in our codebase. The one difference I can think of is that it might specify that I don't care what the function arguments are, however I could get that same effect with something like
```
(fn [& _] (clj->js {:access_token auth-token}))
```
or even simply
```
#(clj->js {:access_token auth-token})
```
Though if that's the only difference, I might even have a slight preference for how it's written because this function is mimicking an existing API where the function does not expect arguments. Thoughts?

Reply:
In general, the codebase itself is not a very good indicator of what's idiomatic or best practice. We're getting better, but only Mohs was an expert on this tech stack when we started, and even his opinions have evolved.

My (current) opinion is that if a function takes an argument but we don't care about it (for example, `onClick`s and other things that receive an event object), then acknowledge that via `(fn [_] ... )`. I also like `constantly` for no-arg functions, but that's kind of a new thing so it's not in much of the codebase yet. `constantly` is nice because it avoids a `#` macro, which is only bad because they can't be nested.

Reply:
Exactly what Matt said. I like `constantly` lately because it communicates clearly that arguments are not used and not intended to be used to construct the return value.

Reply:
Thanks Putnam. That's very helpful for my learning! Sounds like `constantly` is the way to go here as Mohs originally suggested.